maestro/rpm/files/pelion-base-config.yaml

```diff
@@ -8,10 +8,6 @@ scratchPath: /tmp/maestro/scratch
 clientId: "{{ARCH_SERIAL_NUMBER}}"
 network:
   disable: true
-platform_readers:
-  - platform: "fsonly"
-    params:
-      identityPath: "/var/lib/pelion/edge_gw_config/identity.json"
```

Comment:
Is `identity.json` still needed for anything? `wait-for-pelion-identity.service` blocks till this file is created.